### PR TITLE
add stable tests for foreman-tasks

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
@iNecas ACK? :point_up:

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman-tasks.yaml [hunk 1]
```diff
@@ -4,5 +4,9 @@
     branch:
       - master:
           foreman_branch: develop
+      - 0.15.x:
+          foreman_branch: 1.22-stable
+      - 0.16.x:
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman-tasks.yaml]: Nit: I think it's more logical to sort newest -> oldest or oldest -> newest but this feels a bit inconsistent.

**Comment by iNecas** [comment]
@adamruzicka ACK? :point_up:

**Comment by adamruzicka** [comment]
Makes sense, ACK

**Pull request merged**